### PR TITLE
feat(feedback): new feedback analytic event

### PR DESCRIPTION
**Pull request by ryan953**
Adds analytic event, and logic to modify has_new_feedbacks project flag on first non-crashreport feedback ingestion.

This is modeled from: https://github.com/getsentry/sentry/pull/59216

See also:
- https://github.com/getsentry/sentry/pull/61567
- https://github.com/getsentry/sentry/pull/61571

Relates to https://github.com/getsentry/sentry/issues/60711

### CODE REVIEW

File: src/sentry/feedback/usecases/create_feedback.py [hunk 1]
```diff
@@ -168,6 +168,12 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
     if not project.flags.has_feedbacks:
         first_feedback_received.send_robust(project=project, sender=Project)
 
+    if (
+        source is not FeedbackCreationSource.CRASH_REPORT_EMBED_FORM
```
Comment (JoshFerge) [inline src/sentry/feedback/usecases/create_feedback.py:172]: i would modify this to be either only checking that it is `NEW_FEEDBACK_ENVELOPE` instead, as many legacy users using old APIs also send feedbacks via the user report API, or envelope.

File: src/sentry/receivers/onboarding.py [hunk 2]
```diff
@@ -259,6 +260,19 @@ def record_first_feedback(project, **kwargs):
     )
 
 
+@first_new_feedback_received.connect(weak=False)
+def record_first_new_feedback(project, **kwargs):
+    project.update(flags=F("flags").bitor(Project.flags.has_new_feedbacks))
```
Comment (JoshFerge) [inline src/sentry/receivers/onboarding.py:265]: assume its 500 because this is against master and needs the migration for this line to work

**Review by JoshFerge — APPROVED** [review]

File: src/sentry/feedback/usecases/create_feedback.py [hunk 3]
```diff
@@ -168,6 +168,12 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
     if not project.flags.has_feedbacks:
         first_feedback_received.send_robust(project=project, sender=Project)
 
+    if (
+        source is FeedbackCreationSource.NEW_FEEDBACK_ENVELOPE
```
Comment (JoshFerge) [inline src/sentry/feedback/usecases/create_feedback.py:172]: would prefer `==` to `is` here as we generally restrict `is` comparisons to Singleton values like `None` etc. both valid, but i think we more consistently don't use `is` operator. https://stackoverflow.com/a/25858661

Comment (ryan953) [reply src/sentry/feedback/usecases/create_feedback.py:172]:
```suggestion
        source == FeedbackCreationSource.NEW_FEEDBACK_ENVELOPE
```

File: src/sentry/feedback/usecases/create_feedback.py [hunk 4]
```diff
@@ -168,6 +168,16 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
...
         first_feedback_received.send_robust(project=project, sender=Project)
 
+    if (
+        source
+        in [
+            FeedbackCreationSource.NEW_FEEDBACK_ENVELOPE,
+            FeedbackCreationSource.NEW_FEEDBACK_DJANGO_ENDPOINT,
+        ]
+        and not project.flags.has_new_feedbacks
+    ):
```
Comment (ryan953) [inline src/sentry/feedback/usecases/create_feedback.py:178]: cc @JoshFerge I added `NEW_FEEDBACK_DJANGO_ENDPOINT` into here as well

Comment (JoshFerge) [reply src/sentry/feedback/usecases/create_feedback.py:178]: cool, sounds good. plan is to eventually deprecate that django endpoint but helpful to have here while it isn't yet.

**Pull request merged**